lib/ukdebug/outf.h

```diff
@@ -56,7 +56,7 @@ struct out_dev {
 		/* OUTDEV_KERN, OUTDEV_DEBUG */
 		struct {
 			int lvl; /* OUTDEV_KERN only */
-			const char *libname;
+			__u16 libid;
```

Comment:
When building along with lib-musl: `error: unknown type name ‘__u16’`

Reply:
Good point, will fix. An include is then only missing.